### ISSUE TITLE
Collapse-o-matic and printing

### ISSUE THREAD

**Issue by ryanlebreton-gmu** [jmacario-gmu/gmuj-mason-wordpress-theme-twentytwenty-child#21]
Not urgent, but logging this for future consideration: 

Currently if you print a page that has collapse-o-matic content in the collapsed state, the content remains hidden. If you expand it first, then print, it is displayed in the print view. Perhaps this is how we want it to behave, but it would probably be better UX to have it appear in the print view without that extra "expand all" step. 

Interestingly, the "Print, PDF & Email by PrintFriendly" plugin does expand the collapse-o-matic items automatically. 

In any case, we might want to give some attention to the collapse-o-matic markup. If it's questionable for print, it may have some accessibility issues as well. 

Sample page: https://stgehsgmu.wpengine.com/physical-hazards/

**Comment by ryanlebreton-gmu** [comment]
As I think about this more, I find myself on the fence about what the desired behavior should be when printing. But on the markup side, I found out that collapse-o-matic shortcode accepts a "tag" attribute, which would let us wrap collapsible section titles in whichever tag we choose, e.g. h3 or h4, which would be better than the default, which I believe is just a div. 

https://plugins.twinpictures.de/plugins/collapse-o-matic/documentation

This might be a separate issue, but I just wanted to make sure it was recorded somewhere.

**Comment by jmacario-gmu** [comment]
Yes, I am also unsure about what the desired behavior should be. For example, I can imagine many cases where one wouldn't want to print everything in every expandable area on a page, perhaps if they only wanted one or more particular sections. At this point I'd lean toward leaving it as it is; that way the user has more control and has the option of expanding only what they want to print. A what-you-see-is-what-you-get type thing. 

That is interesting that that plugin has an integration with Collapse-o-Matic.

And yes we should definitely test the collapseable sections with screen readers to make sure it's working as it should.
